client/components/course-search-result.jsx

```diff
@@ -27,7 +28,11 @@ export default class CourseSearchResult extends React.Component {
             </div>
           </div>
           <button
-            onClick={() => this.setViewAndCourseId('myLessons', courseId)}
+            onClick={() => {
+              this.setViewAndCourseId('myLessons', courseId);
+              this.props.setBackPage(backPage);
+            }
+            }
```

Comment:
Have the onClick only call one function and call setBackPage() in the setViewAndCourseId() method. Then just rename that method with a clear/succinct name.

Reply:
Took everything in the callback code block and put it into new method 'setNextPage'